process/factory/shard/intermediateProcessorsContainerFactory.go

```diff
@@ -66,16 +68,20 @@ func NewIntermediateProcessorsContainerFactory(
 	if check.IfNil(args.EnableEpochsHandler) {
 		return nil, process.ErrNilEnableEpochsHandler
 	}
+	if check.IfNil(args.TxExecutionOrderHandler) {
```

Comment:
same here, not really needed as it is forwarded

Reply:
if check.IfNil(args.TxExecutionOrderHandler) {